packages/api/cms-api/src/dam/files/files.resolver.ts

```diff
@@ -80,10 +80,10 @@ export function createFilesResolver({ File, Scope: PassedScope }: { File: Type<F
         @SkipBuild()
         async moveDamFiles(
             @Args("fileIds", { type: () => [ID] }) fileIds: string[],
-            @Args("targetFolderId", { type: () => ID, nullable: true }) targetFolderId: string | null | undefined,
+            @Args("targetFolderId", { type: () => ID, nullable: true }) targetFolderId: string | null,
```

Comment:
This change is not correct: Not specifying the `targetFolderId` arg leads to it being undefined. We still need to handle this case. But what should we do then? Move to the root, or do nothing?

Reply:
Yes, that's right. I opted to throw an error if `targetFolderId` is `undefined` because for the `moveDamFiles` operation that is an invalid input

Reply:
The more cleaner way would be to create a new MoveDamFileArgs class an use the `@IsNullable()` decorator, what do you think?

Reply:
Yes true, I extracted the args to MoveDamFileArgs